_posts/2020-09-25-status-update.md

```diff
@@ -0,0 +1,86 @@
+
+- Thanks to [jcpunk][jcpunk] for the tidy up effort to resolve `puppet-lint` and `yamllint` warnings across the following repos:
+  - [`puppetlabs-concat#660`][puppetlabs-concat-pr-660]
+  - [`puppetlabs-kubernetes#438`][puppetlabs-kubernetes-pr-438]
+  - [`puppetlabs-stdlib#1127`][puppetlabs-stdlib-pr-1127]
+  - [`puppetlabs-translate#93`][puppetlabs-translate-pr-93]
+- [`puppetlabs-postgresql#1187`][puppetlabs-postgresql-pr-1187]: Thanks to [ekohl][ekohl] for the enhancement that allows you to remove config entries by setting them to `undef` - a very useful addition!
+- [`puppetlabs-strings#252`][puppetlabs-strings-pr-252]: Thanks to [kozl][kozl] for the various enhancements and fixes in this. Check out [the description](https://github.com/puppetlabs/puppet-strings/pull/252#issue-487952635) for more details.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-accounts`][puppetlabs-accounts] (`6.3.0`)
+
+## Semi-Automated Blog Writing
+
+Well, the way the world is going, everything will be automated soon, won't it?
+This week's blog post was partially written by a new script that will compile:
+
+- The modules released since last week's update
+- The awesome contributions from you, in the Community, that have landed in to `puppetlabs:main`
+
+These are sections that are there every week and the previous process to compiling everything in to Markdown, was a becoming a tedious, manual one.
+So like all repetitive, manual stuff, we automated it! :smiley:
+
+If you're interested in how it all works, check out [`iac#114`][iac-pr-114] and/or the [README][iac-bootstrap-blog-tools].
+
+And before I continue I would just like to give a personal thanks to our very own [Ciaran][sanfrancrisko] for getting this up and working, cutting at least an hour of work from the writing of this post.
+
+## CI Improvements
+
+Over the last few months or so work has been put in to improve our CI coverage with this short message from our very own [Daniel Carabas][carabasdaniel] explaining more:
+
+```
+The IAC Team is currently working on a CI improvement that will increase the testing coverage across all our supported modules and allow contributors to check and run unit and acceptance tests on more platforms. Please look forward to more details about our improved CI in the upcoming weeks.
+```
+
+## New Blogs
+
+We've got several new blog posts out for your viewing pleasure:
+
+- First from [Michael Lombardi][michaeltlombardi] we have an excellent blog which (spoiler alert!) announces the official release of our brand new module [Puppet.Dsc][dsc-repo]! Please check it out [here]({% link _posts/2020-09-21-dsc-release.md %}).
+- Next from [Adrian Iurca][adrianiurca] we have a lovely blog post detailing some imortant changes that were recently made to the [docker][puppetlabs-docker] module. Just click the following link to take a [look]({% link _posts/2020-09-23-docker-params-change-detection %}).
```

Comment:
- Next from [Adrian Iurca][adrianiurca] we have a lovely blog post detailing some important changes that were recently made to the [docker][puppetlabs-docker] module. Just click the following link to take a [look]({% link _posts/2020-09-23-docker-params-change-detection %}).
```suggestion
- Next from [Adrian Iurca][adrianiurca] we have a lovely blog post detailing some imortant changes that were recently made to the [docker][puppetlabs-docker] module. Just click the following link to take a [look]({% link _posts/2020-09-23-docker-params-change-detection %}).
```